react/features/base/popover/components/Popover.web.tsx

```diff
@@ -304,7 +304,8 @@ class Popover extends Component<IProps, IState> {
             && !this.props.overflowDrawer
             && this._contextMenuRef
             && this._contextMenuRef.contains
-            && !this._contextMenuRef.contains(event.target as Node)) {
+            && !this._contextMenuRef.contains(event.target as Node)
+            && !this._containerRef?.current?.contains(event.target as Node)) {
```

Comment:
Why is this necessary?

Reply:
Did you set the Popover `trigger` prop to `click`?

Reply:
yes